<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoGenerator</class>
 <widget class="QDialog" name="OknoGenerator">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>80</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>80</width>
    <height>60</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QToolButton" name="RodzajeSygnalu">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rodzaj Sygnału</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmplitudaNapis">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Amplituda</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Amplituda">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="WypelnienieNapis">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Wypełnienie</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Wypelnienie">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="OkresNapis">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Okres</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Okres">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="ZatwierdzenieUstawien">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ZatwierdzenieUstawien</sender>
   <signal>rejected()</signal>
   <receiver>OknoGenerator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZatwierdzenieUstawien</sender>
   <signal>accepted()</signal>
   <receiver>OknoGenerator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
